cage.sh

```
@@ -312,7 +316,7 @@ seed_home() {
     info "Seeding home directory from $seed_dir"
     $DOCKER cp "$seed_dir/." "$name:/tmp/cage-seed"
     $DOCKER start "$name"
-    $DOCKER exec "$name" sh -c 'cp -rn /tmp/cage-seed/. /home/vscode/ && rm -rf /tmp/cage-seed'
+    $DOCKER exec -u root "$name" sh -c 'chown -R "$(stat -c %u:%g /home/vscode)" /tmp/cage-seed && cp -rp --update=none /tmp/cage-seed/. /home/vscode/ && rm -rf /tmp/cage-seed'
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)